src/assets/styles/layout.css

```diff
@@ -22,6 +22,7 @@ article .asides aside > section {
   color: #fff;
 }
 main {
+  display: block;/* IE 11 gives auto/no height to main */
```

Comment:
This wouldn't be a problem if you'd stop testing in IE11.  I like to save that for a couple days after the release!